### DISCUSSION TITLE
Make the navigation keys (e.g Home, End) work on fields edit

### DISCUSSION THREAD

**Discussion by cs1m0n** [directus/directus#16994] [Feature Requests]
[The navigation keys](https://support.content.office.net/en-us/media/0321edda-f107-c6db-f1dd-9551408d999f.jpg) (e.g. Home, End) don't always work. An example is when duplicating a field on a collection you get the input field where you type in the new name of the field. Clicking Home / End / etc does not do anything.
It would be useful to have them working for faster editing of the existing name.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
